solana-programs/payment-router/Anchor.toml

```diff
@@ -1,15 +1,15 @@
 [features]
 seeds = false
 skip-lint = false
-[programs.localnet]
+[programs.mainnet]
 payment_router = "6pca6uGGV5GYKY8W9aGfJbWPx4pe5mW8wLaP9c3LUNpp"
 
 [registry]
 url = "https://api.apr.dev"
 
 [provider]
-cluster = "Localnet"
-wallet = "/Users/saliou/.config/solana/id.json"
+cluster = "mainnet"
```

Comment:
not sure if i know how this pattern works -- should we similarly have staging / dev versions?

Reply:
well right now this only works for mainnet. testnet/devnet wouldn't help us, unless we did what we briefly talked about wrt creating a devnet raydium pool and using devnet for both swap and wormhole, which we're not doing rn